components/brave_rewards/resources/ui/components/grantWrapper/style.ts

```diff
@@ -32,9 +35,12 @@ export const StyledWrapper = styled<StyleProps, 'div'>('div')`
   flex-wrap: wrap;
   overflow: hidden;
   width: 100%;
-  padding: 0 52px 20px;
+  padding: 0 20px 20px;
```

Comment:
could you please tell me why this was changed? Because of it design was changed.

How it should be with 52px
![image](https://user-images.githubusercontent.com/9574457/75654435-05ac1180-5c60-11ea-89c8-12c8bda10080.png)

with 20px 
![image](https://user-images.githubusercontent.com/9574457/75654463-13619700-5c60-11ea-8c82-c33f57f29909.png)

Reply:
Thanks for the catch! I'll see if the padding change can be reverted.

Is there an easy way to make this "Oops" message appear? The only way I found was to temporarily hack the render code to always display the GrantError component.

Reply:
yes you click claim. Then you disconnect internet and solve captcha